schema/partner.js

```diff
@@ -16,6 +16,9 @@ let PartnerType = new GraphQLObjectType({
   name: 'Partner',
   fields: () => ({
     cached: cached,
+    _id: {
+      type: GraphQLString
+    },
```

Comment:
I'm on board if you wanna just add this to everything. There should maybe be a 'defaults' type that includes `_id`, `id` and `cached` to everything.